Source/WebCore/css/CSSSelector.cpp

```diff
@@ -1042,6 +1047,13 @@ bool CSSSelector::hasExplicitNestingParent() const
     auto checkForExplicitParent = [] (const CSSSelector& selector) {
         if (selector.match() == CSSSelector::Match::NestingParent)
             return true;
+        else if (selector.match() == CSSSelector::Match::UnknownToSerialize) {
```

Comment:
Could we instead compute this information when parsing the selector? Maybe have two flavors of `UnknownToSerialize`?

Reply:
Doesn't seem like we are out of bits.

Reply:
Indeed, we still have plenty of space. Patch now uses a specific enum member for "nest containing".